core/templates/dev/head/components/score-ring/score-ring.directive.ts

```diff
@@ -49,6 +50,26 @@ oppia.directive('scoreRing', [
               setScore(newScore);
             }
           });
+
+          ctrl.getScoreRingColor = function() {
+            if (ctrl.isTestPassed()) {
+              // return color green when passed.
+              return 'rgb(0, 150, 136)';
+            } else {
+              // return color orange when failed.
+              return 'rgb(217, 92, 12)';
+            }
+          };
+
+          ctrl.getScoreOuterRingColor = function() {
+            if (ctrl.isTestPassed()) {
+              // return color green when passed.
+              return 'rgb(143, 217, 209)';
```

Comment:
Is this green too? I see 2 different RGB values commented as green.

Reply:
It is shallow green for the outer ring. I also created a constant for it.